src/utilities/homePageUtilities.js

```diff
@@ -0,0 +1,116 @@
+import { renderPopup, renderComments } from './commentsPopUpUtilities.js';
+import { postCommentData, likeShow } from './postData.js';
+import reservationPopup from './renderPopup.js';
+import { countShows, countDisplayedShows } from '../counterFunctions/showsCounter.js';
+
+const url1 = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/kUJtIKt0WlDGnehZIL7s/comments';
+
+export const getShowsData = async (url) => {
+  const result = await fetch(url);
+  return result;
+};
+
+export function createShowCard(obj, body, likesURL, showsArray) {
+  const div = document.createElement('div');
+  div.classList.add('show-card');
+  div.setAttribute('id', obj.id);
+  div.innerHTML = `
+        <div class='img-placeholder'>
+        <img src='${obj.image.original}' alt='${obj.name} poster'>
+        </div>
+        <div class='interactions-section'>
+          <div class='info-btns'>
+            <button class='reservation-btn'>Reservations</button>
+            <button class='comments'>Comments</button>
+          </div>
+          <div class='like-section'>
+          <button class='like-btn'><i class='fa-regular fa-heart' ></i></button>  
+          <span class='likes-number'></span>
+          </div>
+        </div>
+      `;
```

Comment:
- [ ] as part of the requirement, you guys should stick with ES6 as you did with most of your functions, kindly update `function createShowCard` to be an arrow function 🙏